packages/nextjs/components/AppComponent.tsx

```diff
@@ -1,21 +1,20 @@
+"use client";
```

Comment:
If we use this `use client` here... Wouldn't that make the whole app part of the client bundle? 

cc @rin-st @technophile-04

Reply:
Nope 🙌, 

Since we are slotting it as `children`  : 
```
 <main className="relative flex flex-col flex-1">{children}</main>
```

`children` can be RSC 🙌

Reply:
yep, `AppComponent` doesn't know about children
https://nextjs.org/docs/app/building-your-application/rendering/composition-patterns#supported-pattern-passing-server-components-to-client-components-as-props

Reply:
Amazing, thank you <3